app/logic/serviceLearningCoursesData.py

```diff
@@ -12,6 +12,10 @@
 from app.models import DoesNotExist
 from app.logic.createLogs import createAdminLog
 from app.logic.fileHandler import FileHandler
+from flask import session, g
+import re as regex
+from openpyxl import load_workbook
+from app.logic.term import addPastTerm
```

Comment:
We need to fix our import order here. Built-in libraries first, 3rd party libraries second, our application last.